packages/components/select/src/select.vue

```diff
@@ -30,13 +30,13 @@
           @mouseleave="inputHovering = false"
         >
           <div
-            v-if="multiple"
+            v-if="multiple || isIOS"
```

Comment:
**Question:** I don't know why this is related with the platform, this is not even the dropdown field.

Reply:
hello, this issue is cause by the iOS system that prohibits the `focus` event triggered by non-user actions or asynchronous triggered. You can get more info by these following: 
1. https://github.com/ant-design/ant-design-mobile/issues/5256
2. https://stackoverflow.com/questions/6287478/mobile-safari-autofocus-text-field
![image](https://user-images.githubusercontent.com/30046649/218269208-342e7838-0e9b-4770-a102-2920149e09fa.png)

the `readonly` attribute like this picture1 is a `computed` that change by `visible` and other variable(like picture2).
![image](https://user-images.githubusercontent.com/30046649/218269555-47f5abd5-6595-4fc5-b601-2de10da464ba.png)
![image](https://user-images.githubusercontent.com/30046649/218269648-a6752289-b8ee-448e-8531-10f56bfa5b2b.png)

if `readonly===true` that we can't focus the input element, so when we focus it must be a `false` value, as well as the framework is update async. So the focus will fail.

so, I just fix this problem by letting users click the native input to pull up the keyboard.